docs/pages/docs/04-querying-data/insert-queries.md

```diff
@@ -199,8 +196,8 @@ qb.match(
   var("p2").has("name", "Fuchsia Groan")
 ).insert(
   var()
-    .rel("spouse", "p1")
-    .rel("spouse", "p2")
+    .rel("spouse1", "p1")
+    .rel("spouse2", "p2")
```

Comment:
@haikalpribadi as per our discussion on this, I went ahead and used a unified `spouse` role in all example and test schemas. 

This, however, is causing the last line in `testDifferentMarriageVariants()` test to fail. That's because we get duplicates when running `match $rel (spouse: $x, spouse: $y) isa marriage; get;`

I'm not aware of any way to remove duplicates from a match query at this points, so what I had to do to have this test pass, was to change it from:
`assertEquals(marriageInstancesWithRPsandRoles.size(), 2 * noOfMarriageRelations);`
to
`assertEquals(marriageInstancesWithRPsandRoles.size(), noOfMarriageRelations);`